app/assets/stylesheets/darkswarm/_shop-inputs.scss

```diff
@@ -117,6 +117,13 @@ button.bulk-buy-add.variant-quantity {
   }
 }
 
+/* Hide number arrows on Chrome, Safari, Edge, Opera */
+.variant-quantity::-webkit-outer-spin-button,
+.variant-quantity::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
```

Comment:
For firefox, seems to have equivalent CSS: `-moz-appearance` ; did you try it?

Reply:
Yeah that already exists in the code. That's why there is no issue with firefox :)

Reply:
ahaha, I should have read the file (I only see the diff)